ocs_ci/deployment/deployment.py

```diff
@@ -106,6 +106,7 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
         """
         self.ocp_deployment = self.OCPDeployment()
         self.ocp_deployment.deploy_prereq()
+        self.ocp_deployment.download_installer()
```

Comment:
@gklein ,

I see that https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/aws.py#L351 is trying to copy openshift-install binary in deploy_prereq.

Reply:
Good catch! I'll have to verify AWS UPI too.
I guess we can move this logic under `download_installer()` of the `AWSUPI` `OCPDeployment` implementation